src/api/app/helpers/webui/buildstatus_helper.rb

```diff
@@ -0,0 +1,63 @@
+module Webui::BuildstatusHelper
+  # rubocop:disable Style/AlignHash
+  PACKAGE_STATUS_LIST = {
```

Comment:
Sure that this should live in a helper? For me it looks like Buildresult or Package model is more appropriate?